lib/carto_metadata.js

```diff
@@ -778,85 +760,6 @@ module.exports = function(redis_opts) {
         });
     };
 
-    // DEPRECATED. Used only until Auth fallback in use
-    // AUTH_FALLBACK
-    function isValidApikey(apikey) {
-        return apikey.type !== null &&
-            apikey.user !== null &&
-            apikey.databasePassword !== null &&
-            apikey.databaseRole !== null;
-    }
-
-    // DEPRECATED. Used only until Auth fallback in use
-    // AUTH_FALLBACK
-    function addGrantsFieldsToUserMetadataApikeys(apikey) {
-        apikey.grantsMaps = true;
-        apikey.grantsSql = true;
-
-        return apikey;
-    }
-
-    // DEPRECATED. Used only until Auth fallback in use
-    // AUTH_FALLBACK
-    me.createFauxMasterApikeyFromUserMetadata = function (username, apikeyToken, callback) {
-        var dbParams = {
-            id: 'user_id',
-            database_password: 'database_password',
-            map_key: 'map_key',
-            database_host: 'dbhost',
-            database_name: 'dbname'
-        };
-
-        this.getMultipleUserDBParams(username, dbParams, function(err, params){
-            var apikey = {};
-
-            if (err) {
-                return callback(null, {});
-            }
-
-            if (params.map_key === apikeyToken) {
-                apikey.user_id = params.user_id;
-                apikey.databaseRole = null;
-                apikey.databasePassword = params.database_password;
-                apikey.user = username;
-                apikey.type = 'master';
-
-                addGrantsFieldsToUserMetadataApikeys(apikey);
-            }
-
-            return callback(null, apikey);
-        });
-    };
-
-    // DEPRECATED. Used only until Auth fallback in use
```

Comment:
That comment was key. As we marked as deprecated from the beginning we don't need to bump a major release when removing that function. 

Well done!